appsignals-tests/images/jdbc/src/main/java/software/amazon/opentelemetry/AppController.java

```diff
@@ -46,18 +46,29 @@ public void prepareDB() {
     logger.info("Application Ready");
   }
 
-  @GetMapping("/success")
+  @GetMapping("/success/CREATE_DATABASE")
   @ResponseBody
-  public ResponseEntity<String> success() {
+  public ResponseEntity<String> successCreateDatabase() {
+    try {
+      jdbcTemplate.execute("create database testdb2");
+    } catch (Exception ex) {
+      return ResponseEntity.badRequest().body("failed");
```

Comment:
just throw the exception, the framework will take care of crafting the correct response

Reply:
can fix it while going to next update